Header header
int16 force_front_outer
int16 force_front_inner
int16 force_rear
int16 imu_quat_x
int16 imu_quat_y
int16 imu_quat_z
int16 imu_quat_w
float32 imu_accel_x
float32 imu_accel_y
float32 imu_accel_z
float32 imu_rate_x
float32 imu_rate_y 
float32 imu_rate_z
float32 velocity_cmd_rear
float32 velocity_cmd_front
uint8 skate_fault
float32 position_filt_rear
float32 position_filt_front
float32 velocity_filt_rear
float32 velocity_filt_front
bool dead_man_enable
float32 controller_target_rear
float32 controller_target_front
byte skate_mode
int16 debug_int1
int16 debug_int2
int16 debug_int3
int16 debug_int4
float32 debug_float1
float32 debug_float2
